### PR TITLE
Fix for #84 issues

### PR DESCRIPTION
**Pull request by CAMOBAP795**
- https://github.com/riboseinc/asciidoctor-bibliography/issues/84

### CODE REVIEW

**Comment by CAMOBAP795** [comment]
@ronaldtse I can help with fixing `.rubocop.yml` just let me know. I have checked the problem in outdated `.oss-guides.rubocop.yml`

**Comment by ronaldtse** [comment]
@CAMOBAP795 please also help with Rubocop. Thanks!

**Comment by CAMOBAP795** [comment]
@ronaldtse codeclimat conflicts with rubocop about 'trailing spaces' which behavior we should fixate? https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas

File: spec/fixtures/database-issue84.bib [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+@article{baran_distributed_1964,
+	title = {On distributed communications networks},
+	volume = {12},
+	issn = {0096-1965},
+	number = {1},
+	journal = {IEEE transactions on Communications Systems},
+	author = {Baran, Paul},
+	year = {1964},
+	pages = {1--9},
+	file = {Bar64.pdf:/Users/reza/Zotero/storage/9T733JXT/Bar64.pdf:application/pdf}
```
Comment (ronaldtse) [inline spec/fixtures/database-issue84.bib]: Please replace path `/Users/reza/Zotero/...` with `https://www.rand.org/pubs/papers/P2626.html`.

File: spec/fixtures/database-issue84.bib [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	doi = {10.1109/MAHC.2010.11},
+	language = {en},
+	number = {3},
+	urldate = {2019-02-20},
+	journal = {IEEE Annals of the History of Computing},
+	author = {Lukasik, Stephen},
+	month = mar,
+	year = {2011},
+	pages = {4--21},
+	file = {Lukasik - 2011 - Why the Arpanet Was Built.pdf:/Users/reza/Zotero/storage/I4F9Y8VI/Lukasik - 2011 - Why the Arpanet Was Built.pdf:application/pdf}
```
Comment (ronaldtse) [inline spec/fixtures/database-issue84.bib]: Remove path `/Users/reza/Zotero/storage/I4F9Y8VI/Lukasik ...`

File: spec/fixtures/database-issue84.bib [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+@inproceedings{roberts_arpanet_1986,
+	title = {The {Arpanet} and computer networks},
+	isbn = {0-89791-176-8},
+	publisher = {ACM},
+	author = {Roberts, Larry},
+	year = {1986},
+	pages = {51--58},
+	file = {p51-roberts.pdf:/Users/reza/Zotero/storage/IKUSGSA8/p51-roberts.pdf:application/pdf}
```
Comment (ronaldtse) [inline spec/fixtures/database-issue84.bib]: Remove `/Users/reza/Zotero/storage/IKUSGSA8/p51-roberts.pdf`

File: spec/fixtures/database-issue84.bib [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+@article{dommering_internet:_2015,
+	title = {The {Internet}: a global free space with limited state control},
+	shorttitle = {The {Internet}},
+	url = {https://dare.uva.nl/search?identifier=8672fb24-0825-4bdb-98ac-989e370446ae},
+	language = {en},
+	urldate = {2019-02-20},
+	author = {Dommering, E. and van Ginkel, B. and de Goede, M. and Koops, B. J. and Plooij-van Gorsel, E. and Verrijn Stuart, H. and Smallenbroek, J.},
+	year = {2015},
+	file = {THE INTERNET A GLOBAL FREE SPACE WITH LIMITED STAT.pdf:/Users/reza/Zotero/storage/MQJYNM6X/THE INTERNET A GLOBAL FREE SPACE WITH LIMITED STAT.pdf:application/pdf}
```
Comment (ronaldtse) [inline spec/fixtures/database-issue84.bib]: Remove `/Users/reza/Zotero/storage/MQJYNM6X/THE INTERNET A GLOBAL FREE SPACE WITH LIMITED STAT.pdf`

File: spec/fixtures/database-issue84.bib (hunk 1 above)
Comment (CAMOBAP795) [reply spec/fixtures/database-issue84.bib]: Fixed

File: spec/fixtures/database-issue84.bib (hunk 2 above)
Comment (CAMOBAP795) [reply spec/fixtures/database-issue84.bib]: Fixed

File: spec/fixtures/database-issue84.bib (hunk 3 above)
Comment (CAMOBAP795) [reply spec/fixtures/database-issue84.bib]: Fixed

File: spec/fixtures/database-issue84.bib (hunk 4 above)
Comment (CAMOBAP795) [reply spec/fixtures/database-issue84.bib]: Fixed

**Comment by CAMOBAP795** [comment]
> @ronaldtse codeclimat conflicts with rubocop about 'trailing spaces' which behavior we should fixate? https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas

FYI have changed `Style/TrailingCommaInHashLiteral` rule in rubocop

PR is ready for review again

**Comment by ronaldtse** [comment]
Thank you @CAMOBAP795 !

**Pull request merged**